### PR TITLE
Astro sections

### PR DESCRIPTION
**Pull request by omar-h-omar**

### CODE REVIEW

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -12,17 +12,6 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           reporter: github-pr-review
-  spelling:
```
Comment (omar-h-omar) [inline .github/workflows/reviewdog.yml:15]: @joe2k01 I'm removing this as it seems to be polluting the PRs with useless comments than actually helping.

File: readers-frontend/src/pages/index.astro [hunk 2]
```diff
@@ -28,5 +32,14 @@ const featuredArticlesIds: featuredHome | null =
       <BannerEdition />
     </div>
     <Muse />
+    <div class="px-[3%] pt-8 sm:px-[0.5%] 2xl:px-[13%]">
+      {
+        categories.map((category) => (
+          <div class="featured-section">
```
Comment (joe2k01) [inline readers-frontend/src/pages/index.astro:38]: Since this class doesn't exist, wouldn't it be better to give each section an ID instead?

Comment (omar-h-omar) [reply readers-frontend/src/pages/index.astro:38]: I'm using it for the tests to check that only one featured section is showing

Comment (joe2k01) [reply readers-frontend/src/pages/index.astro:38]: You can do that with an ID

Comment (omar-h-omar) [reply readers-frontend/src/pages/index.astro:38]: do you mean featured-section as an ID or news, comment,..etc for ids?

Comment (joe2k01) [reply readers-frontend/src/pages/index.astro:38]: like news-section, comment-section etc

Comment (omar-h-omar) [reply readers-frontend/src/pages/index.astro:38]: Okay let me have a look into it

**Review by joe2k01 — DISMISSED** [review]

**Review by joe2k01 — APPROVED** [review]

**Pull request merged**